### DISCUSSION TITLE
Direct SDK adding feature request

### DISCUSSION THREAD

**Discussion by PrakharDoneria** [Sketchware-Pro/Sketchware-Pro#489] [Q&A]
This is a feature request from Developers that if possible add a feature through which we can directly add SDK of 3rd party application to out app. Till now we can't send in app notification using firebase if we will have SDK adding option we can easily add latest SDK version to make apps more powerful.

**Comment by j-px** [comment]
Local libraries already exist. You can add external dependencies that way.
